modules/reference/pages/software/software-package-search.adoc

```diff
@@ -1,31 +1,46 @@
 [[ref.webui.channels.search]]
 = Package Search
 
-The [guimenu]``Package Search`` page allows you to search through packages using various criteria provided by the [guimenu]``What to search for`` selection list:
-
-* [guimenu]``Free Form`` -- a general keyword search useful when the details of a particular package and its contents are unknown.
-* [guimenu]``Name Only`` -- Targeted search to find a specific package known by name.
-* [guimenu]``Name and Summary`` -- Search for a package or program which might not show up in the respective package name but in its one-line summary.
-* [guimenu]``Name and Description`` -- Search package names and their descriptions.
-
-image::channels_search_channels.png[scaledwidth=80%]
-
-The [guimenu]``Free Form`` field additionally allows you to search using field names that you prepend to search queries and filter results by that field keyword.
-
-For example, if you wanted to search all of the {sle} packages for the word `java` in the description and summary, type the following in the [guimenu]``Free Form`` field:
+The menu:Software[Package Search] section allows you to search all packages.
+
+Enter your search term in the [guimenu]``Search For`` field.
+
+[[what-search-options]]
+[cols="1,1", options="header"]
+.What to Search Options
+|===
+| Option               | Description
+| Free form            | Performs a general search.
+Use keywords with this option to perform more specific searches.
+| Name only            | Searches only in the names of packages.
+| Name and Summary     | Searches within the name and one-line summary of packages.
+| Name and Description | Searches within names and long descriptions of packages.
+|===
+
+Check the [guimenu]``Channels relevant to your systems`` option to search only channels available for your existing clients.
+Check the [guimenu]``Specific channel you have access to`` option to search within a specific channel.
+Check the [guimenu]``Packages of a specific architecture`` to search only for a particular hardware architecture.
+
+You can perform more specific searches by using keywords in the [guimenu]``Search For`` field and selecting the [guimenu]``Free Form`` option.
+
+[[keyword-search-options]]
+[cols="1,1,1", options="header"]
+.Keyword Search Options
+|===
+| Keyword     | Description | Example
+| name        | Search package names | ``name:SUSE``
+| version     | Search for a package version | ``version:15``
+| filename    | Search within package file names | ``filename:sles``
+| description | Search within the long description | ``description:java``
+| summary     | Search within the one-line summary | ``summary:java``
+| arch        | Search for a package architecture | ``arch:x86_64``
+|===
+
+For example, if you wanted to search all {sle} packages that include `java` in the description and the summary, use this search:
```

Comment:
```suggestion
For example, if you want to search all {sle} packages that include `java` in the description and the summary, use this search:
```
but maybe there is a reason why past tense is better here.

Reply:
nope, should be present. Good spot!